src/firebase.js

```diff
@@ -73,10 +73,8 @@ const getFirestoreConfig = (studyID, docName) => {
 const firestoreConfig = async (studyID, participantID) => {
   const pConfig = await getFirestoreConfig(studyID, participantID);
   const defaultConfig = await getFirestoreConfig(studyID, "default");
-  if (pConfig) {
-    return pConfig;
-  } else if (defaultConfig) {
-    return defaultConfig;
+  if (pConfig || defaultConfig) {
+    return {...defaultConfig, ...pConfig};
```

Comment:
I think for this to work you need to change line 61 to return `{}` instead of false.  But also JS is weird so maybe this is fine?  

Testing it out...

JS is totally weird and this works fine.  Maybe throw a comment on this as it's not the most intuitive that splatting false works.